CSharpFundamentals/Set1-Task2/Set1-Task6/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿int[] player1Coin = new int[5] { 7, 5, 1, 25, 30 };
+int[] player2Coin = new int[5] { 5, 5, 10, 1, 25 };
+double player1TotalScore = 0;
+double player2TotalScore = 0;
+
+for (int i = 0; i < player1Coin.Length;)
+    for (int j = 0; j < player2Coin.Length;)
+    {
+        if (player1Coin[i] > player2Coin[j])
+        {
+            player1TotalScore += 1;
+            Console.WriteLine("\n player one coin is : = " + player1Coin[i]);
+            Console.WriteLine(" player two coin is : = " + player2Coin[i]);
```

Comment:
Aren't these two lines repeated?

Reply:
![image](https://user-images.githubusercontent.com/92961542/171307096-58f392ad-d6c6-45e8-bcf0-09f4d586eb12.png)

this how my screen looks